features/step_definitions/category_steps.rb

```diff
@@ -0,0 +1,26 @@
+Given("the following categories exists") do |table|
+  table.hashes.each do |category|
+    create(:category, category)
+  end
+end
+
+Given("the following article are assigned to categories") do |table|
+  table.hashes.each do |article|
+    category = Category.find_by(name: article[:category])
+    article = Article.find_by(title: article[:title])
+    article.categories.push category
+  end
+end
```

Comment:
Why not do this in the step definition where you create articles?

Reply:
This made more since to us. didn't know if that would mess up all the other test that are using that step so we decided to make a new one.